onnxruntime/test/providers/cpu/reduction/reduction_test_cases.inl

```diff
@@ -4,5374 +4,6180 @@
 #pragma optimize ("", off)
 #elif defined(__GNUC__)
 #if defined(__clang__)
-    #pragma clang optimize off
+	#pragma clang optimize off
 #else
-    #pragma GCC push_options
-    #pragma GCC optimize ("O0")
+	#pragma GCC push_options
+	#pragma GCC optimize ("O0")
 #endif
 #endif
 ReductionTestCases testcases = {
-    // input_data
```

Comment:
is this large diff just formatting?

Reply:
This .inl file is auto generated from the python script. I think the formatting changes came from the 't' to '\t' change in line 81 there.